gravitee-apim-repository/gravitee-apim-repository-mongodb/src/test/java/io/gravitee/repository/mongodb/MongoTestRepositoryConfiguration.java

```diff
@@ -35,13 +38,21 @@
 @EnableMongoRepositories
 public class MongoTestRepositoryConfiguration extends AbstractRepositoryConfiguration {
 
+    private static final Logger LOG = LoggerFactory.getLogger(MongoTestRepositoryConfiguration.class);
+
+    @Value("#{environment.mongoVersion}")
+    private String mongoVersion = "4.4.6";
```

Comment:
```suggestion
    @Value("#{environment.mongoVersion:4.4.6}")
    private String mongoVersion;
```

Reply:
When I'm trying to use this syntax I face:
```shell
Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'mongoTestRepositoryConfiguration': Unsatisfied dependency expressed through field 'mongoVersion'; nested exception is org.springframework.beans.factory.BeanExpressionException: Expression parsing failed; nested exception is org.springframework.expression.spel.SpelParseException: EL1041E: After parsing a valid expression, there is still more data in the expression: 'colon(:)'
Caused by: org.springframework.beans.factory.BeanExpressionException: Expression parsing failed; nested exception is org.springframework.expression.spel.SpelParseException: EL1041E: After parsing a valid expression, there is still more data in the expression: 'colon(:)'
Caused by: org.springframework.expression.spel.SpelParseException: EL1041E: After parsing a valid expression, there is still more data in the expression: 'colon(:)'
```

I tried to use `''` around the value but it's not better  🤔

Reply:
Ok, got it, it should be a `$` instead of a `#`, I did a wrong copy paster form the jdbc part